lib/classes/PluginManager.js

```diff
@@ -215,6 +220,38 @@ class PluginManager {
     );
   }
 
+  getLocalPluginsPaths() {
+    const fs = require('fs');
+
+    const pluginsPaths = this.localPluginsPaths.map(({ entryFileRealPath, originalPath }) => {
```

Comment:
I would name `entryFileRealPath` as `resolvedPath` and `originalPath` as `inputPath`